core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -71,64 +72,129 @@ object ConsumerGroupCommand {
       }
     } catch {
       case e: Throwable =>
-        println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
+        consumerGroupOutputWriter.printError("Executing consumer group command failed due to " + e.getMessage
+            + "\n" + Utils.stackTrace(e))
     } finally {
       consumerGroupService.close()
     }
   }
 
+  case class ConsumerGroupAssignment(group: String, topicOpt: Option[String], partitionOpt: Option[Int], offsetOpt: Option[Long],
+                                     lagOpt: Option[Long], memberIdOpt: Option[String], memberHostOpt: Option[String],
+                                     clientIdOpt: Option[String], logEndOffset: Option[Long])
+
+  trait OutputWriter {
+    def printError(msg: String)
+    def printConfirmation(msg: String)
+    def printAssignmentHeader()
```

Comment:
I agree with merging those three methods. I was just curious about naming convention for the method and the variables used to implement them. Maybe [this line](https://github.com/apache/kafka/pull/1336/files#diff-f1b330624af520e6f9824bd308e21f24R105) helps, where I use `assignments` as the full list of assignments, and then each individual member is an `assignment`. I thought I could use this convention across the board. This is very minor and not a big deal though if it's confusing.